report_generator/run_siege_tests.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+SEED=$1
```

Comment:
you should use tests before this...
$# is the argument count so:

if [ $# -lt 2 ]; then
  echo "Script usage: ... <seed> <concurrent>
  exit 1
fi

-lt is the less than operator:
http://www.tldp.org/LDP/abs/html/comparison-ops.html